src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+}
```

Comment:
They should have same font-family and different font-weight

Reply:
Done.